ui/src/components/btn/QBtn.json

```diff
@@ -15,7 +19,7 @@
     "percentage": {
       "type": "Number",
       "desc": "Percentage (0.0 < x < 100.0); To be used along 'loading' prop; Display a progress bar on the background",
-      "examples": [ 23 ],
+      "examples": [23],
```

Comment:
More space around is generally better for the eyes. Not important though, but I see this change everywhere.